motionless.py

```diff
@@ -247,6 +250,10 @@ def __init__(self, lat=None, lon=None, zoom=None, size_x=400, size_y=400,
         self.region = region
         self.path = []
         self.contains_addresses = False
+        if simplify_threshold_meters is None:
+            self.simplify_threshold = 0
```

Comment:
Why do you need to catch for ``None`` here? It doesn't look like it's needed, as right now the default becomes ``1e-5``, which is the default in GPolyEncode anyway.